src/main.rs

```diff
@@ -15,12 +22,12 @@ fn main() {
         .window_mode(window_mode)
         .window_setup(window_setup)
         .build()
-        .expect("aieee, could not create ggez context!");
+        .expect("Hã!? could not create ggez context!");
```

Comment:
Very important change.

Reply:
I capitalized "Could" in the next commit. Will put out a Rev2 for the meme.